data/102/063/191/102063191.geojson

```diff
@@ -18,12 +24,6 @@
     ],
     "lbl:latitude":47.606032,
     "lbl:longitude":9.886632,
-    "misc:deu_x_preferred_longname":[
-        "Lindau Landkreis"
```

Comment:
The placetype comes before the name in German, so in this case we were storing a bad label value (and bad property prefix). This was fixed and moved to the correct label property.